webapp/app.py

```diff
@@ -23,6 +23,16 @@
 from webapp.feeds import get_rss_feed
 from webapp.doc_parser import FastDocParser
 
+from http.client import responses
+
+from urllib import request, error
+from yaml import load
+
+try:
+    from yaml import CLoader as Loader
+except ImportError:
+    from yaml import Loader
```

Comment:
Is [LibYAML](https://pyyaml.org/wiki/LibYAML) definitely available in the production environment here? Because if not, we should just straightforwardly use the python native loader.

If it is available, is there any way to make it available in local development too? It's not ideal to have code be able to run in 2 distinct ways like this, depending on the running environment. It can lead to subtle bugs.

Reply:
I *believe* it is, I've added [PyYAML](https://pypi.org/project/PyYAML/) to the requirements but if there's a better way of doing this let me know. AFAIK just importing Loader from yaml works so I can reduce it to that if need be

Reply:
If all you're doing is installing pyyaml then you don't have libyaml. libyaml is C bindings, pyyaml is just a python library. If all you're doing is including pyyaml in requirements.txt then you might as well remove the `cloader` line 'cos it won't ever work.

Reply:
That's fine then, I'll remove that